### PR TITLE
📝 Add Word Types Chapter

### PR DESCRIPTION
**Pull request by Benjamin-Piper**
To build locally, please change chapter on line 2 of `03_word_types.mdx` to 2.

Blocked by #1 for styling

Please see the commit descriptions

### CODE REVIEW

File: src/content/chapters/03_word_types.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+   1. Or follow it with a medial consonant pair and go back to step 2.
+
+Examples:
+
+- pcien: property for "_ is bread".
+- ckual: property for "_ is a planet".
+- TODO INSERT TRANSITIVE: 
+- gli: relation for "_ is happy that _ is true".
+
+### Compounds
```
Comment (mia-entropy) [inline src/content/chapters/03_word_types.mdx]: You should explain borrowings first, then tell what can appear inside compounds (root, particles, and a borrowing in first position only)

File: src/content/chapters/03_word_types.mdx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- "a" marks a three-word compound: `acepobu` (property for "is a dictionary").
+- "o" marks a four-or-more compound. Currently, none exist.
+
+Because we don't know how long a four-or-more compound could be, they must be
+ended with `'o` or ` o`.
+
+### Borrowings
+
+Borrowings let you import foreign words or names. They are not present in the
+dictionary, but can appear in compounds combined with a root word.
```
Comment (mia-entropy) [inline src/content/chapters/03_word_types.mdx]:
```suggestion
dictionary, but can appear in compounds combined with a root word or particle.
```

File: src/content/chapters/03_word_types.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+then the word-boundary-marker must be appended to separate it from the prefix.
+
+**Borrowing Structure:**
+
+1. Prefix the word with `u`, and append `'` if the first letter will be `u`.
+1. Begin it with one of the following:
+   1. A single letter (yes this includes h)
+   1. An initial or medial pair
+   1. A consonant triplet
+1. Optionally continue alternating between vhowels and consonant pairs/triplets.
```
Comment (mia-entropy) [inline src/content/chapters/03_word_types.mdx]: Sonorants and single consonants are also allowed in this position.

File: src/content/chapters/03_word_types.mdx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+- Either C1C2 is a medial pair and C2C3 is an initial pair.
+- Or C1 is a sonorant and C2C3 is an initial pair.
+
+Examples:
+
+- eufransestan: property for "_ is France".
+- euinglicban: property for "_ is the English language".
+- za u'unicef: property for "_ is named UNICEF".
+
+Borrowings are terminated with the particle `be`, but this is only required when
```
Comment (mia-entropy) [inline src/content/chapters/03_word_types.mdx:151]: Maybe give more details about that. 2 consecutive borrowings not separated with `be` merge together:
- za udjon ufordo = E is named John Ford
- za udjon be ufordo = E is named John and is "a ford" (maybe the car brand?).

File: src/content/chapters/03_word_types.mdx [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+C1C2C3 is a **consonant triplet** if:
+
+- Either C1C2 is a medial pair and C2C3 is an initial pair.
+- Or C1 is a sonorant and C2C3 is an initial pair.
+
+Examples:
+
+- eufransestan: property for "_ is France".
+- euinglicban: property for "_ is the English language".
```
Comment (mia-entropy) [inline src/content/chapters/03_word_types.mdx]: Borrowing must be ended with ' or space inside compounds too, as ufransestan is a valid borrowing. 
```suggestion
- eufranse'stan: property for "_ is France".
- euinglic'ban: property for "_ is the English language".
```

File: src/content/chapters/03_word_types.mdx [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+[visual&nbsp;parser](https://eberban.github.io/eberban/web/parser_box_glosser.html)!
+
+### Freeform Variables
+
+Freeform variables let you define predicates with a custom name. For example, a
+freeform variable may be used to define a concept in a fictional world. Freeform
+variables do not appear in the dictionary at all.
+
+Freeform variables follow the same structure as borrowings except that their
+prefix is "i". Here's an example:
```
Comment (mia-entropy) [inline src/content/chapters/03_word_types.mdx]: Add that they don't merge together and thus cannot be terminated with `be`, which would collide with usage of `be` at the end of an argument list.

Comment (Benjamin-Piper) [reply src/content/chapters/03_word_types.mdx]: I'll mention `be` used as an argument list terminator in the bindings section.

**Comment by Benjamin-Piper** [comment]
Henceforth I'll be merging without approvals. It's easier to read the finished website and then make a PR anyway.

**Pull request merged**